client/src/modules/reports/generate/systemUsageStat/systemUsageStat.config.js

```diff
@@ -0,0 +1,70 @@
+angular.module('bhima.controllers')
+  .controller('systemUsageStatController', SystemUsageStatController);
+
+SystemUsageStatController.$inject = [
+  '$sce', 'NotifyService', 'BaseReportService',
+  'AppCache', 'reportData', '$state',
+];
+
+function SystemUsageStatController($sce, Notify, SavedReports, AppCache, reportData, $state) {
+  const vm = this;
+  const cache = new AppCache('systemUsageStat');
+  const reportUrl = 'reports/finance/systemUsageStat';
+
+  vm.previewGenerated = false;
+  vm.reportDetails = {
+    shouldShowRemainDetails : 1,
```

Comment:
What is with all these options?  I think you copied and pasted this code from somewhere.  Please go through this code and remove everything you don't need/use.